src/test/java/LoginTests.java

```diff
@@ -18,4 +20,30 @@ public static void LoginEmptyEmailPasswordTest () {
         Assert.assertEquals(driver.getCurrentUrl(), url);
         driver.quit();
     }
-}
+
+    @Test
+    public static void  RegisterToKoel() {
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example `public static void  registerToKoel()`